Header header
int32 num_humans
OpenPoseHuman[] human_list